src/schema/updates/manifest.json

```diff
@@ -21,6 +21,7 @@
             "properties": {
               "required": {
                 "type": "array",
+                "minItems": 1,
```

Comment:
thought: maybe we should briefly mention in #5671 (the issue tracking fixing the underlying issue that forced us to copy this entire part of the schema here in the schema updates) that if we are removing this workaround then we should not forget that we need a `minItems` property to still be applied on the JSONSchema for the `required` property.

To be fair at least of the test added in this patch would fail, and so we can't really miss it, if we decide to add a mention it would be mainly to remind it to ourselves to not have to hit the failure and have to look into why).